sympy/core/basic.py

```diff
@@ -837,6 +837,12 @@ def subs(self, *args, **kwargs):
         else:
             raise ValueError("subs accepts either 1 or 2 arguments")
 
+        if isinstance(self, sympy.core.symbol.Symbol) and isinstance(args[0], sympy.physics.vector.vector.Vector):
```

Comment:
`sympy` isn't imported into either the function scope or the file scope. Look at line 818 and 819. There people have imported things from `sympy` that they needed. You should do that in this case as well. In general, writing `something.something.something.something.something` etc... is hard to read, and a sign that that function/class should be imported into the scope. To prevent circular imports in the core (where this file is), these imports sometimes have to happen inside the function.